sage-android/Sage/app/src/main/java/blueprint/com/sage/shared/adapters/models/AbstractUserListAdapter.java

```diff
@@ -79,6 +84,35 @@ public void onClick(View view) {
             }
         });
         user.loadUserImage(mActivity, viewHolder.mImage);
+
+        int role = user.getRole();
+        if (user.getDirectorId() != 0) {
+            role = 3;
+        }
+        viewHolder.mUserType.setVisibility(View.VISIBLE);
+        viewHolder.mBorder.setVisibility(View.VISIBLE);
+        viewHolder.mUserType.setTypeface(null, Typeface.BOLD);
+        GradientDrawable shape = (GradientDrawable) viewHolder.mUserType.getBackground();
+        viewHolder.mUserType.setText(ROLES[role]);
+        switch(role) {
```

Comment:
i would recommend moving this entire method to like a `UserUtils` file or something, since you set the background color similarly